src/Examples/other_examples.jl

```diff
@@ -37,9 +37,9 @@ function cooketriplet(::Type{T} = Float64, detpix::Int = 1000) where {T<:Real}
         DataFrame(
             SurfaceType = ["Object", "Standard", "Standard", "Standard", "Stop", "Standard", "Standard", "Image"],
             Radius = [Inf, 26.777, 66.604, -35.571, 35.571, 35.571, -26.777, Inf],
-            OptimizeRadius = [false, true, true, true, true, true, true, false],
+            # OptimizeRadius = [false, true, true, true, true, true, true, false],
```

Comment:
These columns can be reintroduced later when the Optimization code is more mature. But they shouldn't be used within the AxisymmetricOpticalSystem constructor. Instead, we should have some kind of AxisymmetricOptimization constructor which parses the extra columns.